packages/react/src/DateInput/DateInput.test.tsx

```diff
@@ -38,4 +38,31 @@ describe('Date input', () => {
 
     expect(ref.current).toBe(component)
   })
+
+  // invalid state
```

Comment:
Is such a comment necessary? The `it` test cases should be self-documenting.

Reply:
The reason for the comment is to imply grouping, i.e. the next n tests are about the invalid state.

It makes more sense to use nested `describe`s for that though, I'll change it